apps/picsa-tools/monitoring-tool/src/app/schema/forms/schema_V4.ts

```diff
@@ -0,0 +1,55 @@
+import type { IPicsaCollectionCreator } from '@picsa/shared/services/core/db_v2';
+import { RxJsonSchema } from 'rxdb';
+
+import { IMonitoringForm_v1, SCHEMA_V1 } from './schema_v1';
```

Comment:
nit(blocking): typo - should use  `schema_v3` so that you don't lose any of the changes from that version